discover/cuda_common.go

```diff
@@ -45,20 +45,19 @@ func cudaVariant(gpuInfo CudaGPUInfo) string {
 		}
 	}
 
-	// Check GPU compute capability FIRST
-	isOldGPU := gpuInfo.computeMajor < 7 || (gpuInfo.computeMajor == 7 && gpuInfo.computeMinor < 5)
-	if isOldGPU {
-		// GPU is Pascal or older (CC <= 7.4) - use CUDA v12 (supports CC 6.1)
-		return "v12"
+	// Check GPU compute capability FIRST, lowest common denominator if multi-gpu
+	for _, gpuInfo := range gpuInfos {
+		if gpuInfo.computeMajor < 7 || (gpuInfo.computeMajor == 7 && gpuInfo.computeMinor < 5) {
+			// GPU is Pascal or older (CC <= 7.4) - use CUDA v12 (supports CC 6.1)
+			return "v12"
+		}
 	}
 
 	// GPU is Turing or newer (CC >= 7.5) - can use newer CUDA
```

Comment:
This was a little bit hard to read. So if _all_ of the cards are greater than or equal to 7.5 then warn if DriverMajor is < 13. Would there be a case where the DriverMajor was different for any of the cards?